LibreNMS/OS/Routeros.php

```diff
@@ -86,19 +88,51 @@ public function discoverWirelessClients()
         );
     }
 
-    /**
+   /**
      * Discover wireless frequency.  This is in MHz. Type is frequency.
      * Returns an array of LibreNMS\Device\Sensor objects that have been discovered
      *
      * @return array Sensors
      */
     public function discoverWirelessFrequency()
-    {
-        return $this->discoverSensor(
+        {
+        $data = $this->fetchData();
+       
+        $sensors = array();
+        foreach ($data as $index => $entry)  {
+        if ($entry['mtxrWlApFreq'] == null)  {
+            return $this->discoverSensor(
             'frequency',
-            'mtxrWlApFreq',
-            '.1.3.6.1.4.1.14988.1.1.1.3.1.7.'
+            'mtxrWl60GFreq',
+            '.1.3.6.1.4.1.14988.1.1.1.8.1.6.'
         );
+        }
+       
+        else{
+
+         $this->discoverSensor(
+                'frequency',
+                'mtxrWlApFreq',
+                '.1.3.6.1.4.1.14988.1.1.1.3.1.7.'
+            );
```

Comment:
A lot of your formatting is now off. Please use four spaces for indentation so it's all aligned correctly. See travis for a list of issues: https://travis-ci.com/librenms/librenms/jobs/151701943#L721